web-services-app/src/main/java/edu/unc/lib/boxc/web/services/processing/IiifV3ManifestService.java

```diff
@@ -178,6 +178,9 @@ private Canvas constructCanvasSection(ContentObjectRecord contentObj) {
         var mimetype = getMimetype(contentObj);
         if (isAudio(mimetype)) {
             setSoundContent(contentObj, paintingAnno, canvas);
+            var audioThumbPath = URIUtil.join(baseAccessPath, "static", "images", "file-audio.png");
+            var thumbnail = new ImageContent(audioThumbPath);
+            canvas.setThumbnails(thumbnail);
```

Comment:
I'd suggest putting this into `setSoundContent` to keep down on the length of `constructCanvasSection`